### ISSUE TITLE
The file & function naming for Dropout is a bit confusing

### ISSUE THREAD

**Issue by jonbakerfish** [NVIDIA-AI-IOT/torch2trt#266]
Why we use `converter/Identity.py` and `def convert_Identity` for the `torch.nn.Dropout` in https://github.com/NVIDIA-AI-IOT/torch2trt/blob/master/torch2trt/converters/Identity.py
 and `convert_identity` for `torch.nn.functional.dropout` in
https://github.com/NVIDIA-AI-IOT/torch2trt/blob/master/torch2trt/converters/identity.py

Is it because dropout returns an identical copy of the input tensor during inference?

**Comment by jaybdub** [comment]
Hi jonbakerfish,

Yep!  That’s correct.  Typically we’ve just introduced new converters only when one that is functionally equivalent is not already available.  In this case the identity converter worked so we registered against that.

Best,
John
